src/docfx/build/markdown/BuildMarkdown.cs

```diff
@@ -43,6 +44,7 @@ internal static class BuildMarkdown
                 Id = id,
                 VersionIndependentId = versionIndependentId,
             };
+            Git.Process(model, file, repo);
```

Comment:
>Process [](start = 16, length = 7)

don't modify the model inside Process, please return the necessary metadata and set them to the model in this level